Tests/Helpers/Helpers.swift

```diff
@@ -169,6 +169,28 @@ func deleteInstanceResources(completionHandler: @escaping (TestHelperError?) ->
     }.resume()
 }
 
+func synchronousHTTPRequest(
```

Comment:
If it's synchronous, why not return `(Data?, URLResponse?, Error?)` directly? Having a synchronous method with a completionHandler feels weird.

Reply:
d'oh!  🤦‍♂️